### PR TITLE
More tested projects

### PR DESCRIPTION
**Pull request by mchindri**
Tested project updated to iio_app from #1140

### CODE REVIEW

File: projects/ad9434-fmc-500ebz/src/ad9434_fmc_500ebz.c [hunk 1]
```diff
@@ -163,7 +165,21 @@ int main(void)
...
 	};
 
-	return iio_app_start(&iio_axi_adc_init_par);
+	status = iio_axi_adc_init(&iio_axi_adc_desc, &iio_axi_adc_init_par);
+	if (status < 0)
+		return status;
+
+	iio_axi_adc_get_dev_descriptor(iio_axi_adc_desc, &dev_desc);
+	struct iio_data_buffer read_buff = {
+		.buff = (void *)ADC_DDR_BASEADDR,
```
Comment (amiclaus) [inline projects/ad9434-fmc-500ebz/src/ad9434_fmc_500ebz.c:174]: why not using buffers?

File: projects/ad9739a-fmc-ebz/src/ad9739a_fmc_ebz.c [hunk 2]
```diff
@@ -211,7 +212,22 @@ int main(void)
...
 
-	return iio_app_start(&iio_axi_dac_init_par);
+	status = iio_axi_dac_init(&iio_axi_dac_desc, &iio_axi_dac_init_par);
+	if (status < 0)
+		return status;
+
+	iio_axi_dac_get_dev_descriptor(iio_axi_dac_desc, &dev_desc);
+
+	struct iio_data_buffer read_buff = {
+		.buff = (void *)DAC_DDR_BASEADDR,
```
Comment (amiclaus) [inline projects/ad9739a-fmc-ebz/src/ad9739a_fmc_ebz.c:222]: ditto

File: projects/ad9434-fmc-500ebz/src/ad9434_fmc_500ebz.c (hunk 1 above)
Comment (mchindri) [reply projects/ad9434-fmc-500ebz/src/ad9434_fmc_500ebz.c:174]: This commit is not related to buffer, but indeed that must be changed too.

Comment (amiclaus) [reply projects/ad9434-fmc-500ebz/src/ad9434_fmc_500ebz.c:174]: In this manner, you won't have to test future updated projects twice (once for iio_app update and once for iio buffers) 😄

**Comment by mchindri** [comment]
v2:
 - Add iio buffers instead of raw addresses

File: projects/ad9739a-fmc-ebz/src/ad9739a_fmc_ebz.c [hunk 3]
```diff
@@ -219,8 +221,8 @@ int main(void)
 	iio_axi_dac_get_dev_descriptor(iio_axi_dac_desc, &dev_desc);
 
 	struct iio_data_buffer read_buff = {
-		.buff = (void *)DAC_DDR_BASEADDR,
-		.size = 0xFFFFFFFF,
+		.buff = (void *)dac_buffer,
```
Comment (cristiantpop) [inline projects/ad9739a-fmc-ebz/src/ad9739a_fmc_ebz.c:224]: Why it has to be used in ad9739a_dmac_init_param also? Maybe some future commit is necessary.

Comment (mchindri) [reply projects/ad9739a-fmc-ebz/src/ad9739a_fmc_ebz.c:224]: I forgot :)

**Comment by mchindri** [comment]
v3:
 - Remove changes of Add iio buffers instead of raw addresses . This needs a better look into it

**Review by amiclaus — APPROVED** [review]

**Pull request merged**